plugins/mcreator-core/datalists/sounds.yaml

```diff
@@ -996,3 +997,188 @@
 - entity.zoglin.death
 - entity.zoglin.hurt
 - entity.zoglin.step
+- block.amethyst_block.break
+- block.amethyst_block.chime
+- block.amethyst_block.fall
+- block.amethyst_block.hit
+- block.amethyst_block.place
+- block.amethyst_block.step
+- block.amethyst_cluster.break
+- block.amethyst_cluster.fall
+- block.amethyst_cluster.hit
+- block.amethyst_cluster.place
+- block.amethyst_cluster.step
+- item.axe.wax_off
+- item.honeycomb.wax_on
+- entity.axolotl.attack
+- entity.axolotl.death
+- entity.axolotl.hurt
+- entity.axolotl.idle_air
+- entity.axolotl.idle_water
+- entity.axolotl.splash
+- entity.axolotl.swim
+- block.azalea.break
+- block.azalea.fall
+- block.azalea.hit
+- block.azalea.place
+- block.azalea.step
+- block.azalea_leaves.break
+- block.azalea_leaves.fall
+- block.azalea_leaves.hit
+- block.azalea_leaves.place
+- block.azalea_leaves.step
+- block.big_dripleaf.break
+- block.big_dripleaf.fall
+- block.big_dripleaf.hit
+- block.big_dripleaf.place
+- block.big_dripleaf.step
+- item.bucket.empty_powder_snow
+- item.bucket.fill_axolot

```

Comment:
This one contains a spelling mistake and there are 2 new sounds for axolotls in buckets.
item.bucket.empty_axolotl
item.bucket.fill_axolotl